COPYRIGHT

```diff
@@ -1,3 +1,6 @@
+The code in the "libcperciva/external" directory is distributed
```

Comment:
I'm not completely certain where you want this copyright notice to be.

Reply:
Hmm, tarsnap has a top-level COPYRIGHT and a libcperciva/COPYRIGHT.  But spiped only has a top-level COPYRIGHT.